### DISCUSSION TITLE
Connected everything but display doesn't work

### DISCUSSION THREAD

**Discussion by mattburns** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#634] [Q&A]
Hi, noob here looking for some troubleshooting help with my square-pump Helsinki. Basically, it isn't working. When connected, the spa display is unlit and unresponsive. The blue led is flashing and serving the ui, but interacting with the web interface has no effect (eg, bubbles/pump ui toggle does nothing).

I'm terrible at soldering, but _think_ I managed to get it together...
![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/1316126/6285fb41-f157-4887-bc72-1da7760fbfe2)

I've uploaded the firmware and filesystem image and configured wifi and hardware.
<img width="444" alt="image" src="https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/1316126/04f021c0-c714-48c6-814b-a3b70b328fa7">

Any ideas for what I should test first?

**Comment by visualapproach** [comment]
[try this](http://layzspa.local/hwtestinfo.html)

**Reply by mattburns** [reply]
:'(
```
CIO to DSP pin 3 fail!
CIO to DSP pin 4 fail!
CIO to DSP pin 5 fail!

DSP to CIO pin 3 fail!
DSP to CIO pin 4 fail!
DSP to CIO pin 5 fail!
```

**Reply by mattburns** [reply]
Ah, looks like I put the LLC in the wrong way around...

**Reply by mattburns** [reply]
Huzzah, got it working in the end! Everything appears to work perfectly. Coffees on their way, thanks @visualapproach :)

Helsinki with S100102. Will try setting up home assistant another day... ;)

Cheers, Matt

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/1316126/c3dca96b-1574-418d-bb31-4d43e7de3f8b)

**Reply by visualapproach** [reply]
Awesome! Thank you 🙏🏻 
Happy it worked out for you

**Reply by johnpony** [reply]
Hi so,

I have the exact same output as in your picture and it's not working at all when I connect it to the pump. 

I have no idea what I'm doing so I don't know what it means when you say 'you put the LCC the wrong way'.

Did you get this working?

**Reply by visualapproach** [reply]
> I have no idea what I'm doing so I don't know what it means when you say 'you put the LCC the wrong way'.

LLC = Logic Level Converter = the red module. It must be mounted in the correct orientation. HV side to the right and LV tto the left in the image.
